client/app/cda/details/home/detail/edit/cda-detail-edit.html

```diff
@@ -1,63 +1,40 @@
-    </div>
-    <div class="oui-modal__body oui-modal__body_no-icon">
-        <div class="oui-modal__body-text" data-ng-if="CdaDetailEditCtrl.saving">
-            <div class="text-center">
-                <oui-spinner></oui-spinner>
-            </div>
-        </div>
-        <div class="oui-modal__body-text" data-ng-if="!CdaDetailEditCtrl.saving">
-            <h2 class="oui-modal__title" data-translate="cda_detail_edit_title"></h2>
+<form id="editDetailForm" name="editDetailForm">
+    <oui-modal
+        data-primary-action="editDetailForm.$valid && CdaDetailEditCtrl.editCluster()"
+        data-primary-label="{{ ::'common_modify' | translate }}"
+        data-secondary-action="CdaDetailEditCtrl.closeModal()"
+        data-secondary-label="{{ ::'common_cancel' | translate }}"
+        data-on-dismiss="CdaDetailEditCtrl.closeModal()">
 
-            <label for="oui-label" data-translate="cda_detail_edit_label" class="cda-input-label"></label>
-            <input type="text" class="oui-input"
-                id="label" name="label"
-                data-ng-model="CdaDetailEditCtrl.model.label"
-                data-ng-maxLength="CdaDetailEditCtrl.options.label.maxLength"
-                required />
-            <div class="text-danger" data-ng-messages="editDetailForm.label.$error" data-ng-if="editDetailForm.label.$dirty">
-                <p class="animate-fadeInLeft-enter animate-fadeOutRight-leave" >
-                    <span data-ng-message="required" data-translate="cda_detail_edit_label_required"></span>
-                    <span data-ng-message="maxlength" data-translate="cda_detail_edit_label_maxlength" data-translate-values="{ maxlength: CdaDetailEditCtrl.options.label.maxLength }"></span>
-                </p>
-            </div>
+        <cui-message-container data-messages="CdaDetailEditCtrl.messages"></cui-message-container>
 
-            <label for="crushTunable" data-translate="cda_detail_edit_crushTunable" class="oui-label"></label>
-            <div class="oui-select">
-                <select class="oui-select__input"
-                    id="crushTunable" name="crushTunable"
-                    data-ng-model="CdaDetailEditCtrl.model.crushTunable"
-                    data-ng-options="crushTunableValue | lowercase for crushTunableValue in CdaDetailEditCtrl.crushTunableValues"
-                    required />
-                <i class="oui-icon oui-icon-chevron-down"></i>
-            </div>
-            <div class="text-danger" data-ng-messages="editDetailForm.crushTunable.$error" data-ng-if="editDetailForm.crushTunable.$dirty">
-                <p class="animate-fadeInLeft-enter animate-fadeOutRight-leave" >
-                    <span data-ng-message="required" data-translate="cda_detail_edit_crushTunable_required"></span>
-                </p>
-            </div>
+        <h2 class="oui-modal__title" ng-bind="::'cda_detail_edit_title' | translate "></h2>
+        <div class="text-center">
```

Comment:
oui-modal__spinner class could be interesting to centralize it with a minimum of effort.
It should be identical in every modal for all managers